client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -105,3 +105,7 @@
 	flex: none;
 	margin-left: 16px;
 }
+
+.editor-simple-payments-modal__form .upload-image {
```

Comment:
Yes, it could. But if we don't have any styles for `editor-simple-payments-modal__form` I think it's preferable to write it like this so it's more clear.